src/main/java/com/conveyal/gtfs/stats/FeedStats.java

```diff
@@ -145,6 +154,40 @@ public LocalDate getEndDate() {
 
         return endDate;
     }
+
+    public LocalTime getStartTime (LocalDate date) {
+        return this.pattern.getStartTimeForTrips(getTripsForDate(date));
+    }
+
+    public LocalTime getEndTime (LocalDate date) {
+        return this.pattern.getEndTimeForTrips(getTripsForDate(date));
+    }
+
+    /** Get total revenue time (in seconds) for all trips on a given date. */
+    public int getTotalRevenueTimeForTrips (LocalDate date) {
+        return this.pattern.getTotalRevenueTimeForTrips(getTripsForDate(date));
+    }
+
+    /** Get total revenue distance (in meters) for all trips on a given date. */
+    public double getTotalRevenueDistanceForTrips (LocalDate date) {
+        return this.pattern.getTotalRevenueDistanceForTrips(getTripsForDate(date));
+    }
+
+    /** in seconds */
+    public double getDailyAverageHeadway (LocalDate date, LocalTime from, LocalTime to) {
```

Comment:
I question the value of this metric. It seems completely dependent on your bounds and how the GTFS is coded (e.g. are platforms coded separately...)